### PR TITLE
8328378: Convert java/awt/FileDialog/FileDialogForDirectories test to main

### PR DESCRIPTION
**Pull request by azuev-java**
Convert test to main manual and move to the appropriate folder
Remove old files

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328378](https://bugs.openjdk.org/browse/JDK-8328378): Convert java/awt/FileDialog/FileDialogForDirectories test to main (**Bug** - P4)

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) ⚠️ Review applies to [d8f2aa64](https://git.openjdk.org/jdk/pull/18358/files/d8f2aa645203346a40c150ed5e8dd49549db42c7)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18358/head:pull/18358` \
`$ git checkout pull/18358`

Update a local copy of the PR: \
`$ git checkout pull/18358` \
`$ git pull https://git.openjdk.org/jdk.git pull/18358/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18358`

View PR using the GUI difftool: \
`$ git pr show -t 18358`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18358.diff">https://git.openjdk.org/jdk/pull/18358.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18358#issuecomment-2004629779)

### CODE REVIEW

File: test/jdk/java/awt/FileDialog/FileDialogForDirectories.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import java.awt.Frame;
+import java.lang.reflect.InvocationTargetException;
+import javax.swing.JFrame;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+
+/*
+ * @test
+ * @bug 7161437
+ * @summary We should support "apple.awt.fileDialogForDirectories" property.
```
Comment (prsadhuk) [inline test/jdk/java/awt/FileDialog/FileDialogForDirectories.java:36]: @library is missing...Does it work for you without this? I see most of your latest PRs doesn't have this tag

File: test/jdk/java/awt/FileDialog/FileDialogForDirectories.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ * @requires (os.family == "mac")
+ * @run main/manual FileDialogForDirectories
+ */
+
+public class FileDialogForDirectories {
+    private static Button showBtn;
+    private static FileDialog fd;
```
Comment (prsadhuk) [inline test/jdk/java/awt/FileDialog/FileDialogForDirectories.java:43]: Can be made local in main

Comment (prsadhuk) [reply test/jdk/java/awt/FileDialog/FileDialogForDirectories.java:45]: Still could have been made local...

**Review by prsadhuk — APPROVED** [review]

File: test/jdk/java/awt/FileDialog/FileDialogForDirectories.java (hunk 1 above)
Comment (azuev-java) [reply test/jdk/java/awt/FileDialog/FileDialogForDirectories.java:36]: I had PassFailJFrame.class in my classpath so i could run these tests easier. I fixed all of the headers now.

File: test/jdk/java/awt/FileDialog/FileDialogForDirectories.java (hunk 2 above)
Comment (azuev-java) [reply test/jdk/java/awt/FileDialog/FileDialogForDirectories.java]: Fixed.

**Pull request closed without merging**